### PR TITLE
Add Zookeeper Watcher

### PR DESCRIPTION
**Pull request by craigbr**
Not in docker-compose, yet. Let's get it built first

### CODE REVIEW

File: zookeeper-watcher/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* `0.0.1` - standard [SemVer][1]
+* `latest`: the latest version recommended to use with other Monasca
+  components.
+
+
+Usage
+-----
+
+**zookeeper-watcher** requires a running instance of [Zookeeper][2]. It can be launched using
+command similar to this ```docker run -d -p 9092:9092 -l zookeeper monasca/zookeeper```.
```
Comment (matrixik) [inline zookeeper-watcher/README.md]: What for is `-p 9092:9092`?

```
/ # netstat -nap
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name    
tcp        0      0 127.0.0.11:42423        0.0.0.0:*               LISTEN      -
tcp        0      0 172.18.0.13:44112       172.18.0.6:2181         ESTABLISHED 7/zookeeper-watcher
tcp        0      0 :::8080                 :::*                    LISTEN      7/zookeeper-watcher
udp        0      0 127.0.0.11:53321        0.0.0.0:*                           -
Active UNIX domain sockets (servers and established)
Proto RefCnt Flags       Type       State         I-Node PID/Program name    Path
/ # ps aux
PID   USER     TIME   COMMAND
    1 root       0:00 /sbin/tini -- /start.sh
    7 root       0:00 /zookeeper-watcher
   23 root       0:00 /bin/ash
   32 root       0:00 ps aux
```

File: zookeeper-watcher/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+This parameter can be specified using environment variables:
+
+| Variable                      | Default          | Description                           |
+|-------------------------------|------------------|---------------------------------------|
+| `STAY_ALIVE_ON_FAILURE`       | `false`          | If `true`, container stays alive for 2 hours after zookeeper watcher exits |
+
+
+Additional parameters are available. See https://github.com/monasca/monasca-watchers for more details.
+
+[1]: http://semver.org/
```
[the remark this answers is not in the document]
Comment (craigbr) [reply zookeeper-watcher/README.md]: All of the other READMEs reference http://semver.org/. I'm going for consistency on this one.

File: zookeeper-watcher/README.md (hunk 1 above)
Comment (craigbr) [reply zookeeper-watcher/README.md]: Copy paste error. Fixed

[the remark this answers is not in the document]
Comment (craigbr) [reply zookeeper-watcher/README.md]: Done

**Comment by craigbr** [comment]
@matrixik The different log lines come from the GO Zookeeper library.

**Pull request merged**